templates/taxi/manufacturer_update.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+{% block content %}
+    <h1>{{ object|yesno:"Update, Create" }} Manufacturer</h1>
```

Comment:
The use of `yesno` filter here is not appropriate because this template is specifically for updating a manufacturer. Consider changing the heading to simply 'Update Manufacturer'.